modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/TextStyles.kt

```diff
@@ -143,12 +143,13 @@ fun TextH40(
     fontWeight: FontWeight = FontWeight.W500,
     disableAutoScale: Boolean = false,
     fontScale: Float = 1f,
+    lineHeight: TextUnit = 21.sp.scaled(disableAutoScale, fontScale),
 ) {
     Text(
         text = text,
         color = color,
         fontSize = 15.sp.scaled(disableAutoScale, fontScale),
-        lineHeight = 21.sp.scaled(disableAutoScale, fontScale),
+        lineHeight = lineHeight,
```

Comment:
This change breaks font scaling. Instead it should be `llineHeight: TextUnit = 21.sp` and `lineHeight = lineHeight.scaled(disableAutoScale, fontScale)` so that disabling and font scale is applied correctly.